crates/motionfile/src/motion_interpolator.rs

```diff
@@ -12,19 +13,20 @@ use types::ConditionInput;
 
 #[derive(Debug, Default, Serialize, Deserialize)]
 pub struct ConditionedSpline<T> {
-    pub entry_condition: Option<ConditionType>,
+    pub entry_condition: Option<DiscreteConditionType>,
+    pub enable: Option<Vec<ContinuousConditionType>>,
```

Comment:
Instead of `enable` maybe `spline_conditions` or similar? IMHO `enable` is more confusing than helpful.

Reply:
What do you think of `motion_interrupts`? Since they can occur at any time in a segment to abort/pause the animation.